docker/dockerconfig.json

```diff
@@ -0,0 +1,49 @@
+{
+  "DiskCache": {
+    "cachePath": "./docker-state/DiskCacheData",
+    "localSymbolDirs": [
+    ],
+    "maxSizeMB": 200,
+    "port": 8888,
+    "symbolURLs": [
+      "https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1/"
+    ],
+    "log": {
+      "path": "./docker-state/diskcache.log",
+      "level": 30,
+      "maxFiles": 5,
+      "maxFileSizeMB": 50
+    }
+  },
+  "SymServer": {
+    "port": 8080,
+    "memcachedServers": [
+      "127.0.0.1:11211"
```

Comment:
We're running everything in a single container. Another way to do this is to have a separate memcached container. Since the tests start up and shut down memcached for every test, I figured it's easier to set this up to have everything in the same container.